core/types/rollup_cost.go

```diff
@@ -375,9 +375,9 @@ func NewL1CostFuncFjord(l1BaseFee, l1BlobBaseFee, baseFeeScalar, blobFeeScalar *
 	}
 }
 
-// EstimatedL1Size estimates the number of bytes the transaction will occupy in its L1 batch using
-// the Fjord linear regression model, and returns this value scaled up by 1e6.
-func EstimatedL1SizeScaled(costData RollupCostData) *big.Int {
+// EstimatedDASizeScaled estimates the number of bytes the transaction will occupy in its L1 batch
+// using the Fjord linear regression model, and returns this value scaled up by 1e6.
+func EstimatedDASizeScaled(costData RollupCostData) *big.Int {
```

Comment:
should we make this function internal? the scale it applies is pretty specific to how the scalars work

Reply:
yep, done.